src/utils/PersonUtils.js

```diff
@@ -122,48 +94,10 @@ function getUserProfile(user :any) :UserProfile {
   };
 }
 
-function sortByProfileLabel(user1 :any, user2 :any) {
-
-  const userId1 :string = getUserId(user1);
-  const userId2 :string = getUserId(user2);
-  const label1 :string = getUserProfileLabel(user1) || userId1;
-  const label2 :string = getUserProfileLabel(user2) || userId2;
-
-  if (label1 < label2) {
-    return -1;
-  }
-
-  if (label1 > label2) {
-    return 1;
-  }
-
-  return 0;
-}
-
-function filterUser(user :any, filter :string) :boolean {
-
-  const userId :string = getUserId(user);
-  if (filter === userId) {
-    return true;
-  }
-
-  const auth0UserProfile = get(user, 'profile', user);
-  const nickname :string = get(auth0UserProfile, 'nickname', '');
-  const username :string = get(auth0UserProfile, 'username', '');
-  const email :string = get(auth0UserProfile, 'email', '');
-
-  return (
-    _includes(nickname, filter)
-    || _includes(username, filter)
-    || _includes(email, filter)
-  );
-}
-
 export {
-  filterUser,
   getPrincipalId,
-  getUserId,
+  // getUserId,
   getUserProfile,
   getUserProfileLabel,
-  sortByProfileLabel,
+  // sortByProfileLabel,
```

Comment:
do you need these comments anymore?

Reply:
Nope. I've cleaned it up in another branch.